src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseClient.java

```diff
@@ -280,4 +280,31 @@ public void createCollectionExerciseEvent(EventDTO event) {
               event.getCollectionExerciseId(), event.getTag(), response.getStatus()));
     }
   }
+
+  void sampleSummaryReadiness(final UUID sampleSummaryId) throws CTPException {
+    sampleSummaryReady(sampleSummaryId);
+  }
+
+  private int sampleSummaryReady(final UUID sampleSummaryId) throws CTPException {
```

Comment:
return value is never used

Reply:
Updates in separate PR https://github.com/ONSdigital/rm-collection-exercise-service/pull/321